pyshimmer/dev/calibration.py

```diff
@@ -0,0 +1,89 @@
+# pyshimmer - API for Shimmer sensor devices
+# Copyright (C) 2023  Lukas Magel
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import re
```

Comment:
Can you use some basic PEP8 formatter to format the file? I wrote most of the code in PyCharm, they have some decent basic formatting support. I know that I don't have any clear specification what should be used and tbh yours is the biggest contribution to the project I have had so far.

Reply:
PEP8, got it.